apps/lukso-manager/src/webserver/dist/apps/lukso-status/main.6fd04046714fcaa29b9b.js

```diff
@@ -0,0 +1 @@
sSelector]),Ve(1),lt("ngIf",n[r.peersSelector]),Ve(5),Rr(n.beacon_head_slot)}}function HN(t,e){1&t&&(O(0,"div",14),ne(1," Can not connect to node, is it running? "),k())}let $N=(()=>{class t{constructor(n){this.searchTerm$=new F_(""),this.peersSelector='p2p_peer_count{state="Connected"}';const r=this.searchTerm$.pipe(Fl(o=>o.length>2),j_(10),B_());this.metrics$=n.getMetrics$(),this.filteredMetrics$=k_([r,this.metrics$]).pipe(Ge(([o,s])=>Object.keys(s).filter(i=>i.includes(o)).reduce((i,a)=>Object.assign(i,{[a]:s[a]}),{})),Ge(o=>Object.entries(o)),H_(()=>Pl({})))}}return t.\u0275fac=function(n){return new(n||t)(B(ON))},t.\u0275cmp=to({type:t,selectors:[["lukso-vanguard-status"]],decls:6,vars:4,consts:[[1,"title"],["class","columns",4,"ngIf","ngIfElse"],["loadingOrError",""],[1,"columns"],[1,"column"],[1,"table","is-hoverable"],[1,"property-name"],[4,"ngIf"],["class","tag is-success",4,"ngIf"],["class","tag is-warning",4,"ngIf"],["class","tag is-danger",4,"ngIf"],[1,"tag","is-success"],[1,"tag","is-warning"],[1,"tag","is-danger"],[1,"has-background-danger","has-text-white","p-5"]],template:function(n,r){if(1&n&&(O(0,"h2",0),ne(1,"Vanguard"),k(),et(2,BN,14,3,"section",1),Ic(3,"async"),et(4,HN,2,0,"ng-template",null,2,Pc)),2&n){const o=$u(5);Ve(2),lt("ngIf",Tc(3,2,r.metrics$))("ngIfElse",o)}},directives:[ml],pipes:[Dl],styles:["[_nghost-%COMP%]{display:block;margin:0 20px 40px}.property-name[_ngcontent-%COMP%]{width:200px}"],changeDetection:0}),t})(),UN=(()=>{class t{constructor(n){this.http=n,this.title="lukso-status",this.http=n}updateClient(){this.http.get("/update-client").subscribe()}}return t.\u0275fac=function(n){return new(n||t)(B(Zi))},t.\u0275cmp=to({type:t,selectors:[["lukso-root"]],decls:15,vars:0,consts:[["role","navigation","aria-label","main navigation",1,"navbar","is-dark"],[1,"navbar-brand"],["href","#",1,"navbar-item"],[1,"container","is-max-widescreen"],[1,"message","is-info"],[1,"message-header"],[1,"message-body"],[1,"button","is-info",3,"click"]],template:function(n,r){1&n&&(O(0,"nav",0),O(1,"div",1),O(2,"a",2),ne(3," LUKSO Node Status "),k(),k(),k(),O(4,"div",3),Qu(5,"lukso-pandora-status"),Qu(6,"lukso-vanguard-status"),O(7,"article",4),O(8,"div",5),O(9,"p"),ne(10,"Info"),k(),k(),O(11,"div",6),ne(12," If you just started your node, please give it at least an hour to find peers, p2p discovery takes some time. "),k(),O(13,"button",7),Ju("click",function(){return r.updateClient()}),ne(14," Update Client "),k(),k(),k())},directives:[FN,$N],styles:["[_nghost-%COMP%]{display:block;font-family:sans-serif;margin:0}.container[_ngcontent-%COMP%]{max-width:600px}nav[_ngcontent-%COMP%]{margin-bottom:40px}"]}),t})(),WN=(()=>{class t{}return t.\u0275fac=function(n){return new(n||t)},t.\u0275mod=Tn({type:t,bootstrap:[UN]}),t.\u0275inj=Qt({providers:[],imports:[[jA,cN]]}),t})();(function(){if(fy)throw new Error("Cannot enable prod mode after platform setup.");dy=!1})(),LA().bootstrapModule(WN).catch(t=>console.error(t))}},Jr=>{Jr(Jr.s=621)}]);
```

Comment:
IDK if we should track by git this path `apps/lukso-manager/src/webserver/dist/***`

Reply:
No we should not, it's fixed.